code/game/mecha/combat/phazon.dm

```diff
@@ -7,7 +7,7 @@
 	step_energy_drain = 3
 	max_integrity = 200
 	deflect_chance = 30
-	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 30, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
+	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 30, "bomb" = 90, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)

```

Comment:
90% is a to much, 80% is way better for the glass cannon that is the Phazon

Reply:
no it's not, it has 200 integrity.